python/src/UniformOrderStatistics_doc.i.in

```diff
@@ -0,0 +1,60 @@
+%feature("docstring") OT::UniformOrderStatistics
+"UniformOrderStatistics distribution.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+dimension : int
+    The dimension of the distribution.
+
+See also
+--------
+openturns.MaximumEntropyOrderStatisticsDistribution
+
+Notes
+-----
+This is the joint distribution of the order statistics :math:`(X_{(1)},\dots,X_{(n)})`
+of a sample :math:`(X_1,\dots, X_n)` of :math:`n` independent random variables
+uniformly distributed over :math:`[0,1]`.
+In other words:
+
+- let :math:`\sampleSize \in \Nset` be the sample size ;
+- let :math:`X_i \sim \mathcal{U}(0,1)` for :math:`i \in \set{1, ..., \sampleSize}` be independent random variables ;
+- order them in increasing order: :math:`X_{(1)} \leq X_{(2)} \leq \ldots \leq X_{(\sampleSize)}`.
+
+Its probability density function is defined as (see [arnold2008]_ eq. 2.2.4 page 11):
+
+.. math::
+
+    f_X(x) = n!\,\mathbf{1}_{\cS}(x)
```

Comment:
The notation f_X(x) might be confusing with the classical notation of the PDF of X, should we introduce another notation for the PDF of the order statistics ?

"Its probability density function (noted ......) is defined as (see [arnold2008]_ eq. 2.2.4 page 11):"

Reply:
@l-brevault Thanks for the reference. I changed the notation of the random vector of order statistics using something github is not able to render...